account_invoice_triple_discount/models/line_triple_discount.py

```diff
@@ -84,7 +84,7 @@ def triple_discount_preprocess(self):
         # more digits than allowed from field's precision,
         # so let's increase it just for saving it correctly in cache
         discount_field = self._fields['discount']
-        discount_original_digits = discount_field._digits
+        discount_original_digits = dp.get_precision("Discount")(self.env.cr)
```

Comment:
I think you can avoid this variable `discount_original_digits`... just assign `dp.get_precision("Discount")(self.env.cr)` in https://github.com/SimoRubi/account-invoicing/blob/475829f94890a4deeef92ed7c58165deaeeda751/account_invoice_triple_discount/models/line_triple_discount.py#L103

Reply:
done, thanks!